src/test/common.node.ts

```diff
@@ -328,11 +328,14 @@ export async function captureScreenShot(contextOrFileName: string | Mocha.Contex
             : hashjs
                   .sha256()
                   .update(contextOrFileName.currentTest?.fullTitle() || '')
-                  .digest('hex');
+                  .digest('hex')
+                  .substring(0, 10); // Ensure file names are short enough for windows.
     const testTitle = typeof contextOrFileName === 'string' ? '' : contextOrFileName.currentTest?.title || '';
+    const counter = (screenShotCount.get(fullTestNameHash) || 0) + 1;
+    screenShotCount.set(fullTestNameHash, counter);
     const fileNamePrefix =
         typeof contextOrFileName === 'string' ? contextOrFileName : `${testTitle}_${fullTestNameHash}`;
-    const name = `${fileNamePrefix}_${uuid()}`.replace(/[\W]+/g, '_');
```

Comment:
Removing the guid and reducing the length of the hash string, hopefully this will ensure the file name is short enough for windows.